backend/cypress/e2e/search-tests.cy.js

```diff
@@ -0,0 +1,156 @@
+import searchPage from '../page_objects/search-page.js';
+import searchData from '../fixtures/search_page_data/search_data.json'
+
+let search;
+let yearsToCheck;
+let ueiOrein;
+let aln;
+let name;
+let accDate;
+let state;
+let cogOrover;
+let findings;
+let directFunding;
+let majorProgram;
+let fiscalYearEndMonth;
+let typeRequirement;
+let entityType;
+let passthroughName;
+let reportId;
+
+beforeEach(() => {
+  cy.visit('/dissemination/search/');
+
+  yearsToCheck = searchData.yearsToCheck;
+  ueiOrein = searchData.ueiOrein;
+  aln = searchData.aln;
+  name = searchData.name;
+  accDate = searchData.accDate;
+  state = searchData.state;
+  cogOrover = searchData.cogOrover;
+  findings = searchData.findings;
+  directFunding = searchData.directFunding;
+  majorProgram = searchData.majorProgram;
+  fiscalYearEndMonth = searchData.fiscalYearEndMonth;
+  typeRequirement = searchData.typeRequirement;
+  entityType = searchData.entityType;
+  passthroughName = searchData.passthroughName;
+  reportId = searchData.reportId;
+
+
+  search = new searchPage();
+});
+
+//basic search test
+describe('Test Basic Search Fields', () => {
+
+  it('checks Audit Years', () => {
```

Comment:
We may want a more descriptive name for these tests, since they fill out every box

Reply:
That's a oversight on my end, you're right they do need to be more descriptive.